core/src/test/java/org/testcontainers/utility/RegistryAuthLocatorTest.java

```diff
@@ -16,7 +20,7 @@
 public class RegistryAuthLocatorTest {
 
     @Test
-    public void lookupAuthConfigWithoutCredentials() throws URISyntaxException {
+    public void lookupAuthConfigWithoutCredentials() throws URISyntaxException, IOException {
```

Comment:
Why was `IOException` added to all the throws clauses of the tests?

Reply:
because of this https://github.com/testcontainers/testcontainers-java/pull/6238/files#diff-d2e41605733b1a39c4ec3e5839fbebd71851fd71a6f3b42eab97d61f50989dd4R280